src/AspNetCoreModuleV2/AspNetCore/HandlerResolver.cpp

```diff
@@ -158,11 +160,20 @@ HandlerResolver::GetApplicationFactory(IHttpApplication &pApplication, std::uniq
             }
         }
 
-        m_loadedApplicationHostingModel = pConfiguration.QueryHostingModel();
+        m_loadedApplicationHostingModel = options.QueryHostingModel();
         m_loadedApplicationId = pApplication.GetApplicationId();
-        RETURN_IF_FAILED(LoadRequestHandlerAssembly(pApplication, pConfiguration, pApplicationFactory));
+        RETURN_IF_FAILED(LoadRequestHandlerAssembly(pApplication, options, pApplicationFactory));
 
     }
+    catch(ConfigurationLoadException &ex)
```

Comment:
nit: let's keep naming similar to c#. ex => configurationLoadException?

Reply:
We mostly use ex in C#